controllers/result_test.go

```diff
@@ -37,10 +37,37 @@ var _ = Describe("ResultController", func() {
 					response := MakeAuthenticatedRequest("GET", "/", nil, nil, user)
 					responseBody := GetResponseBody(response)
 
+					Expect(responseBody).To(ContainSubstring("Results (2)"))
 					Expect(responseBody).To(ContainSubstring(result1.Keyword))
 					Expect(responseBody).To(ContainSubstring(result2.Keyword))
 					Expect(responseBody).NotTo(ContainSubstring(result3.Keyword))
 				})
+
+				Context("given the user filter results", func() {
```

Comment:
```suggestion
				Context("given the user filters results", func() {
```

Reply:
✅  updated in [4453fdb](https://github.com/carryall/go-google-scraper-challenge/pull/74/commits/4453fdb1910d3c9347e4a8c3fb079f3fd6e4d58a)